magprime/automated_emails.py

```diff
@@ -19,69 +20,94 @@ def __init__(self, event):
+           ident='magprime_techops_volunteer',
+           sender='MAGFest TechOps <techops-heads@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Chipspace volunteering', 'chipspace.txt',
-           lambda a: a.staffing and a.assigned_to(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
+           lambda a: a.staffing and a.assigned_to(c.CHIPSPACE),
+           ident='magprime_chipspace_volunteer',
+           sender='MAGFest ChipSpace <chipspace@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Chipspace shifts', 'chipspace_trusted.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and a.trusted_in(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and a.trusted_in(c.CHIPSPACE),
+           ident='magprime_chipspace_trusted_volunteer',
+           sender='MAGFest ChipSpace <chipspace@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Chipspace', 'chipspace_untrusted.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and not a.trusted_in(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and not a.trusted_in(c.CHIPSPACE),
+           ident='magprime_chipspace_untrusted_volunteer',
+           sender='MAGFest ChipSpace <chipspace@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Staff Suite rules', 'food_volunteers.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and not a.trusted_in(c.FOOD_PREP), sender='MAGFest Staff Suite <chefs@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and not a.trusted_in(c.FOOD_PREP),
+           ident='magprime_food_untrusted_volunteer',
+           sender='MAGFest Staff Suite <chefs@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest message from Chef', 'food_trusted_staffers.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and a.trusted_in(c.FOOD_PREP), sender='MAGFest Staff Suite <chefs@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and a.trusted_in(c.FOOD_PREP),
+           ident='magprime_food_trusted_volunteer',
+           sender='MAGFest Staff Suite <chefs@magfest.org>')
 
 AutomatedEmail(Attendee, 'Want to help run MAGFest poker tournaments?', 'poker.txt',
-               lambda a: a.has_shifts_in(c.TABLETOP), sender='MAGFest Tabletop <tabletop@magfest.org>')
+               lambda a: a.has_shifts_in(c.TABLETOP),
+               ident='magprime_tabletop_volunteer_poker_inquiry',
+               sender='MAGFest Tabletop <tabletop@magfest.org>')
 
 StopsEmail('MAGFest Staff Support', 'staff_support.txt',
-           lambda a: a.assigned_to(c.STAFF_SUPPORT) and not a.trusted_in(c.STAFF_SUPPORT))
+           lambda a: a.assigned_to(c.STAFF_SUPPORT) and not a.trusted_in(c.STAFF_SUPPORT),
+           ident='magprime_staff_support_volunteer',)
```

Comment:
There's a trailing comma here. I don't know off the top of my head if that's a problem or not.